Core/Packages/PackageInfo.cs

```diff
@@ -4,34 +4,105 @@
 
 namespace NuGetPe
 {
+    /// <summary>
+    /// Encapsulates relevant Nuget package information
```

Comment:
```suggestion
    /// Encapsulates relevant NuGet package information
```

Apologies for the nitpick! 😅